src/ripple/beast/core/SystemStats.cpp

```diff
@@ -61,24 +60,29 @@ getStackBacktrace()
         std::distance(std::begin(stack), std::end(stack)),
         stack, nullptr);
 
-    HeapBlock<SYMBOL_INFO> symbol;
-    symbol.calloc (sizeof (SYMBOL_INFO) + 256, 1);
-    symbol->MaxNameLen = 255;
-    symbol->SizeOfStruct = sizeof (SYMBOL_INFO);
+    // Allow symbols that are up to 1024 characters long.
+    std::size_t const nameLength = 1024;
+    std::size_t const nameSize = nameLength * sizeof(SYMBOL_INFO::Name);
+
+    std::unique_ptr<SYMBOL_INFO, decltype(std::free)*>
+        symbol { std::malloc(sizeof (SYMBOL_INFO) + nameSize), std::free };
```

Comment:
Use a char array buffer to avoid the dynamic allocation?
```
char buffer[sizeof(SYMBOL_INFO) + nameSize];
PSYMBOL_INFO symbol ={static_cast<PSYMBOL_INFO>(buffer)};
```